packages/manager/src/features/Linodes/MigrateLinode/ConfigureForm.tsx

```diff
@@ -98,6 +105,16 @@ export const ConfigureForm = React.memo((props: Props) => {
     [backupEnabled, currentLinodeType]
   );
 
+  const linodeIsInEdgeRegion = useIsEdgeRegion(currentRegion, regions ?? []);
```

Comment:
Some of the utils unnecessary. Can we just do the following?

```suggestion
  const linodeIsInEdgeRegion = currentActualRegion?.site_type === 'edge'
```